Web-Scraping/Medium-Articles-Details-Scrapping/app/__init__.py

```diff
@@ -0,0 +1,108 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import random
+import numpy as np
+import itertools
+import time
+
+class Scrap:
+    """
```

Comment:
@kaustubhgupta it'll be recommended if you shift this comment part to the README as it's description part!